src/spaces/p3.js

```diff
@@ -4,14 +4,10 @@ import sRGB from "./srgb.js";
 
 export default new RGBColorSpace({
 	id: "p3",
+	cssId: "display-p3",
 	name: "P3",
 	base: P3Linear,
 	// Gamma encoding/decoding is the same as sRGB
 	fromBase: sRGB.fromBase,
-	toBase: sRGB.toBase,
-	formats: {
-		color: {
-			id: "display-p3",
-		}
-	},
+	toBase: sRGB.toBase
```

Comment:
Does our linting disallow trailing commas? If so, it shouldn't. I think trailing commas in things like this would really help minimize the diffs.

Reply:
Good call. Right now our linter is agnostic about trailing commas, so I tried to match the pattern I found in the files. But I agree it results in noisy diffs, so I adjusted in https://github.com/color-js/color.js/commit/b1cf95a4a68dd7c54f0a6e62609318be780e9d92.

I also opened #440 to separately consider enforcing trailing commas.